pinot-core/src/main/java/org/apache/pinot/core/data/table/SimpleIndexedTable.java

```diff
@@ -49,12 +50,12 @@
    * Initializes the data structures needed for this Table
    * @param dataSchema data schema of the record's keys and values
    * @param aggregationFunctions aggregation functions for the record's values
```

Comment:
Shouldn't this be replaced with FunctionContext?

Reply:
No. `AggregationFunction` is the execution class instead of the info class. `FunctionContext` will replace `AggregationInfo` in the following PR of replacing BrokerRequest classes in aggregation and transform functions